frontend/tests/Factories/Contest.php

```diff
@@ -103,9 +103,14 @@ class ContestParams {
      * @var null|string
      */
     public $scoreMode;
+    /**
+     * @readonly
+     * @var bool
+     */
+    public $checkPlagiarism;
```

Comment:
For consistency between arguments, could you add a blank line here:
```suggestion

    /**
     * @readonly
     * @var bool
     */
    public $checkPlagiarism;
```
?

Reply:
Sure.